packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbar.md

```diff
@@ -0,0 +1,399 @@
+---
+title: 'DataToolbar'
+cssPrefix: 'pf-c-data-toolbar'
+typescript: true
+propComponents: ['DataToolbar', 'DataToolbarContent', 'DataToolbarItem', 'DataToolbarGroup', 'DataToolbarToggleGroup', 'DataToolbarBreakpointMod', 'DataToolbarSpacer']
```

Comment:
Looks like we need `DataToolbarSpaceItems` on order to apply `.pf-m-space-items{-on-[breakpoint]}` to groups

Reply:
Both spacing of Items and Groups can use the DataToolbarSpacer, it's built to be smart enough to build both -space-items and -spacer, but you're right I was missing the -spacer on Groups, so I'll add a property to groups to specify if you are adding the spacer to the group or to the items in the group.